File: Exercises_C++/1179.cpp
#include <bits/stdc++.h>
using namespace std;

int pares[5];
int impares[5];


int main() {
	
	int countp, counti;
    countp = 0;
    counti = 0;
    int count = 0;
    int numero;
    while (count < 15) {

        cin >> numero;
        count += 1;

        if (numero % 2 == 0) {
            pares[countp] = numero;
            countp += 1;

            if (countp == 5) {
                for (int c = 0 ; c < countp ; c++) {
					cout << "par[" << c << "] = " << pares[c] << endl;
				
				}
				
				countp = 0;
            }

        }
        else {
            impares[counti] = numero;
            counti += 1;
			
			if (counti == 5) {
               for (int k = 0 ; k < counti ; k++) {
                   cout << "impar[" << k << "] = " << impares[k] << endl;

                }

               counti = 0;
            }
        }

        }
		 if (counti != 0) {
            for (int c = 0 ; c < counti ; c++) {
                   cout << "impar[" << c << "] = " << impares[c] << endl;

                }

            }


        if (countp != 0) {
            for (int k = 0 ; k < countp ; k++) {
                   cout << "par[" << k << "] = " << pares[k] << endl;

                }

            }
	}

File: Bee-Crowd/Exercises_Python/1179.py
pares = []
impares = []
for p in range(15):
    numero = int(input())

    if numero % 2 == 0:
        pares.append(numero)
    else:
        impares.append(numero)

    if len(pares) == 5:
        k = 0
        for c in pares:
            print('par[{}] = {}'.format(k,c))
            k += 1
        pares = []

    if len(impares) == 5:
        i = 0
        for t in impares:
            print('impar[{}] = {}'.format(i,t))
            i += 1
        impares = []

s = 0
for r in impares:
    print('impar[{}] = {}'.format(s,r))
    s += 1

u = 0
for o in pares:
    print('par[{}] = {}'.format(u,o))
    u += 1